content/security/team.adoc

```diff
@@ -61,7 +61,7 @@ You can contribute to the security of Jenkins and its plugin ecosystem even with
 * Inform us about plugin security updates without a corresponding security advisory.
   Plugin maintainers may be unaware of our process, so this helps ensure all security updates are properly announced.
 * Document security best practices for Jenkins administrators and Jenkins developers.
-* As a Jenkins developer, develop features and improvements that help admins secure their instance.
+* As a Jenkins developer, develop features and improvements that help admins secure their controller and agents.
```

Comment:
Optional change for consistency with the preceding change that uses "controllers and agents".  Approved whether or not this change is applied.
```suggestion
* As a Jenkins developer, develop features and improvements that help admins secure their controllers and agents.
```